.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: ["2.2", "2.3", "2.4", "2.5", "2.6", "2.7"]
```

Comment:
Testing Ruby 3.0 is important because of the keyword argument semantic change.

Also not sure it's still worth testing super old rubies like 2.4.

Reply:
Absolutely, did you see:

> I suggest merging this to demonstrate the versions which do work, and then modernising the version support to align with the Ruby maintenance policy and the Ruby on Rails maintenance policy.

That would come next:

https://github.com/sj26/marginalia/compare/remove-unsupported-versions...modernise-versions

Reply:
Ah sorry, I didn't finish my coffee. It's up to you, but if it was me I'd go straight to the versions we actually care about.